.flowconfig

```diff
@@ -1,14 +1,7 @@
-# [version]
-# 0.126.1
-#
```

Comment:
I propose that we don't specify a version and let dependabot keep us up-to-date when it can.